soh/soh/z_message_OTR.cpp

```diff
@@ -101,12 +132,9 @@ extern "C" void OTRMessage_Init()
         sStaffMessageEntryTablePtr = (MessageTableEntry*)malloc(sizeof(MessageTableEntry) * file2->messages.size());
 
         for (size_t i = 0; i < file2->messages.size(); i++) {
-            sStaffMessageEntryTablePtr[i].textId = file2->messages[i].id;
-            sStaffMessageEntryTablePtr[i].typePos =
-                (file2->messages[i].textboxType << 4) | file2->messages[i].textboxYPos;
-            sStaffMessageEntryTablePtr[i].segment = file2->messages[i].msg.c_str();
-            sStaffMessageEntryTablePtr[i].msgSize = file2->messages[i].msg.size();
+            SetMessageEntry(sStaffMessageEntryTablePtr[i], file2->messages[i]);
         }
+        OTRMessage_LoadCustom("custom/text/staff_message_data_static/*", sStaffMessageEntryTablePtr, file2->messages.size());
```

Comment:
Same as above, replace `custom/` with `override/`. We're reserving `custom/` for loading non-overriding messages into `CustomMessageTable`s later on.